### PR TITLE
CC-2362 HSTS missing from HTTPS server

### PR DESCRIPTION
**Pull request by romanbakaleyko**

### CODE REVIEW

**Comment by romanbakaleyko** [comment]
retest this

File: web/cpserver.go [hunk 1]
```diff
@@ -148,6 +148,8 @@ func (sc *ServiceConfig) Serve(shutdown <-chan (interface{})) {
 		}
 
 		logger.Debug("Calling CC uiHandler")
+		w.Header().Add("Strict-Transport-Security","max-age=31536000")
```
Comment (jplouis) [inline web/cpserver.go]: Does this need to be applied to public endpoints (vhosts) as well?

Comment (jplouis) [inline web/cpserver.go]: @rphillipsz your input might be helpful since you've reworked parts of this code.

**Comment by romanbakaleyko** [comment]
demo screen shot in the ticket https://jira.zenoss.com/browse/CC-2362

File: web/cpserver.go (hunk 1 above)
Comment (romanbakaleyko) [inline web/cpserver.go]: added requested changes, demo screen shot in the ticket https://jira.zenoss.com/browse/CC-2362

**Pull request merged**